packages/clay-multi-select/src/ClayMultiSelect.js

```diff
@@ -362,13 +360,19 @@ class ClayMultiSelect extends ClayComponent {
 				};
 			})[0];
 
-		item = filteredItem || item;
+		if (this.allowOnlyItemsFromAutocomplete) {
```

Comment:
Is this condition that guarantees that an item will not be created from the input? When I try to create a label through the comma it deletes all the content, I think I would have to keep the text. Probably because of this https://github.com/liferay/clay/blob/2.x-develop/packages/clay-multi-select/src/ClayMultiSelect.js#L115